tests/Microsoft.Bot.Builder.LanguageGeneration.Tests/Examples/Expand.lg

```diff
@@ -100,4 +100,16 @@ They are ${ShowAlarm(alarms[1])}
 [MyStruct
     Speak = ${GetAge()}
     Text = zoo
+]
+
+# PointOfInterestSuggestedActionNameText(Data)
```

Comment:
Maybe you could change the name to make it a more formal test name

Reply:
The name seems ok I think.